From 18bb27ec8c52ef67f4911daea8cb4d07b09978f8 Mon Sep 17 00:00:00 2001
From: Stefan Roscher <ossrosch@linux.vnet.ibm.com>
Date: Fri, 22 Feb 2008 15:33:42 +0100
Subject: [PATCH] ehca: Change version number

this patch changes the version number for ehca, to seperate it from the one in kernel upstream.
It will apply against vlads ofed-1.3 git tree.
Please pick it for OFED-1.3 release.

Signed-off-by: Stefan Roscher <stefan.roscher@de.ibm.com>
---
 drivers/infiniband/hw/ehca/ehca_main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: ofed_kernel/drivers/infiniband/hw/ehca/ehca_main.c
===================================================================
--- ofed_kernel.orig/drivers/infiniband/hw/ehca/ehca_main.c
+++ ofed_kernel/drivers/infiniband/hw/ehca/ehca_main.c
@@ -50,7 +50,7 @@
 #include "ehca_tools.h"
 #include "hcp_if.h"
 
-#define HCAD_VERSION "0026"
+#define HCAD_VERSION "SVNEHCA_0026"
 
 MODULE_LICENSE("Dual BSD/GPL");
 MODULE_AUTHOR("Christoph Raisch <raisch@de.ibm.com>");
